src/app/(contents)/post/PostList.tsx

```
@@ -0,0 +1,170 @@
+                        onClick={() => router.push("/post/register")}
+                    >
+                        <Image src={darkMode ? "/pen_white.svg" : "/pen.svg"} alt={"plus"} width={25} height={25} />
+                    </ButtonBasic>
+                )}
+            </div>
+            <br />
+            <LineBasic />
+            <br />
+            <div></div>
+            <div className={styles.postListWrapper}>
+                {postList.length > 0 ? (
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)